src/cmd/deploy.rs

```diff
@@ -85,8 +90,9 @@ pub fn run(args: &Args) -> Result<()> {
             &format!(
                 r###"
 cros-workon-{board} start {packages}
-~/trunk/src/scripts/update_kernel.sh --remote={} --ssh_port {} --remote_bootargs
+~/trunk/src/scripts/update_kernel.sh {} --remote={} --ssh_port {} --remote_bootargs
 "###,
+                ab_update,
```

Comment:
In Rust, if is an expression (= it can be a value) so you don't have to use a temporary variable here.
```
if args.ab_update { "--ab_update" } else { "" }
```
c.f. https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=0f36cb209de6f27c6d2aecab01ec67ff

Reply:
Got it.